sql/src/test/java/io/crate/planner/operators/WindowAggTest.java

```diff
@@ -66,6 +70,22 @@ public void testTwoWindowFunctionsWithDifferentWindowDefinitionResultsInTwoOpera
         assertThat(plan, isPlan(e.functions(), expectedPlan));
     }
 
+    @Test
+    public void test_window_agg_output_for_select_with_standalone_ref_and_window_func_with_filter() {
+        var plan = plan("SELECT y, AVG(x) FILTER (WHERE x > 1) OVER() FROM t1");
```

Comment:
the test was a bit wrong, it'd always pass.. the scenario where it occurs is again filter function and standalone ref in the select list.